### PR TITLE
Setup coveralls

### PR DESCRIPTION
**Pull request by Siianko**
### PT link
https://www.pivotaltracker.com/story/show/158908174

### User story
As a development team, 
In order to see how much of our code is covered in tests
I would like to use Coveralls as my analysis tool.

### What did we do
- added gem 'coveralls' to Gemfile, 
- ran bundle, 
- updated rspec and cucumber with coveralls config
- put coverage into .gitignore

### CODE REVIEW

**Review by faraznaeem — CHANGES_REQUESTED** [review]
You need to add coverage to the git ignore file remove it from the cache

File: features/support/env.rb [hunk 1]
```diff
@@ -1,5 +1,8 @@
 require 'cucumber/rails'
 
+require 'coveralls'
+Coveralls.wear_merged!('rails')
+
```
Comment (faraznaeem) [inline features/support/env.rb]: these should be in the top of the file

**Review by faraznaeem — CHANGES_REQUESTED** [review]
Please provide a proper description for the PR

**Review by oliverochman — APPROVED** [review]
LGTM!

**Review by faraznaeem — APPROVED** [review]

**Pull request merged**